library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -295,6 +294,23 @@ public void run() {
         });
     }
 
+    /**
+     * Authorize using Session Token.
+     *
+     * @param sessionToken     Session Token
+     * @param listener         The OktaAuthListener to receive callback with results
+     */
+    public void authorize(
```

Comment:
Let's name this `authenticate` to match iOS: https://github.com/okta/okta-sdk-appauth-ios/pull/113#discussion_r250386206